mne_connectivity/spectral/tests/test_spectral.py

```diff
@@ -480,113 +480,102 @@ def test_spectral_connectivity(method, mode):
 
 
 @pytest.mark.parametrize("method", ["cacoh", "mic", "mim", _gc])
-def test_spectral_connectivity_epochs_multivariate(method):
+@pytest.mark.parametrize("n_components", [1, 2, 3, None])
+def test_spectral_connectivity_epochs_multivariate(method, n_components):
```

Comment:
In addition to adding checks for multiple comps in this `spec_conn_epochs` unit test, I went ahead with some other improvements to clean up the code and make it generally more extensive.

Also, the simulated data being used here was a bit limiting to what we could test with multiple comps, so I switched from the old data simulation to the new `make_signals_in_freq_bands()` func, so ticks off some maintenance stuff too.